tautulli/umbrel-app.yml

```diff
@@ -0,0 +1,64 @@
+manifestVersion: 1
+id: tautulli
+category: Media
```

Comment:
we made a recent change where category values are lower case, so this case be changed to `media`